add handler_group to adsorption.py

oxana-a opened the pull request:
## Summary
* added handler_group = "default" to Optimize and TransmuteFW in atomate.vasp.fireworks.core
* added handler_group = "md" to get_slab_fw, get_wf_slab, and get_wfs_all_slabs in atomate.vasp.workflows.base.adsorption

## TODO (if any)
* update atomate.vasp.workflows.tests

JosephMontoya-TRI commented:
This is perfect, and your TODO was the only thing I had to say.  For the updated test, I would just add an assertion to ensure that the handler group is getting passed in invocation,  i. e. that the handler argument is actually in the constructed workflow.

JosephMontoya-TRI commented:
If you'd like, I can do this and send back your way to give you a sense of how the tests and assertions work.  Also happy to let you give it a shot.  Let me know.

oxana-a commented:
I had a quick look at it last week but wasn't sure what to change, I'll
give it another shot and I'll let you know if I cannot figure it out.

Thank you!
Oxana

On Tue, Apr 9, 2019 at 1:57 PM Joseph Montoya <notifications@github.com>
wrote:

> If you'd like, I can do this and send back your way to give you a sense of
> how the tests and assertions work. Also happy to let you give it a shot.
> Let me know.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/montoyjh/atomate/pull/3#issuecomment-481432600>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AQEsuMR3LcY8N-r9nqgnf-wWEON5WRHXks5vfP6egaJpZM4clcxw>
> .
>

JosephMontoya-TRI commented:
Great!  Yeah basically I'd just add some statement which gets the handler_group property from the workflow after it's constructed and ensures that it's actually 'md'.

utf committed:
Merge pull request #629 from mkhorton/patch-9

Small patch to add option to disable/enable Bader analysis on VASP ingestion

janosh committed:
add common powerup add_metadata

janosh committed:
write tests for add_additional_fields_to_taskdocs and add_metadata

janosh committed:
fix typos

dependabot-preview[bot] committed:
Bump numpy from 1.20.2 to 1.20.3

Bumps [numpy](https://github.com/numpy/numpy) from 1.20.2 to 1.20.3.
- [Release notes](https://github.com/numpy/numpy/releases)
- [Changelog](https://github.com/numpy/numpy/blob/main/doc/HOWTO_RELEASE.rst.txt)
- [Commits](https://github.com/numpy/numpy/compare/v1.20.2...v1.20.3)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump boto3 from 1.17.69 to 1.17.73

Bumps [boto3](https://github.com/boto/boto3) from 1.17.69 to 1.17.73.
- [Release notes](https://github.com/boto/boto3/releases)
- [Changelog](https://github.com/boto/boto3/blob/develop/CHANGELOG.rst)
- [Commits](https://github.com/boto/boto3/compare/1.17.69...1.17.73)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump maggma from 0.26.0 to 0.27.0

Bumps [maggma](https://github.com/materialsproject/maggma) from 0.26.0 to 0.27.0.
- [Release notes](https://github.com/materialsproject/maggma/releases)
- [Changelog](https://github.com/materialsproject/maggma/blob/main/docs/CHANGELOG.md)
- [Commits](https://github.com/materialsproject/maggma/compare/v0.26.0...v0.27.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #663 from hackingmaterials/dependabot/pip/boto3-1.17.73

dependabot-preview[bot] committed:
Bump moto from 2.0.6 to 2.0.7

Bumps [moto](https://github.com/spulec/moto) from 2.0.6 to 2.0.7.
- [Release notes](https://github.com/spulec/moto/releases)
- [Changelog](https://github.com/spulec/moto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/spulec/moto/compare/2.0.6...2.0.7)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #662 from hackingmaterials/dependabot/pip/numpy-1.20.3

dependabot-preview[bot] committed:
Merge pull request #666 from hackingmaterials/dependabot/pip/maggma-0.27.0

dependabot-preview[bot] committed:
Merge pull request #665 from hackingmaterials/dependabot/pip/moto-2.0.7

dependabot-preview[bot] committed:
Bump moto from 2.0.7 to 2.0.8

Bumps [moto](https://github.com/spulec/moto) from 2.0.7 to 2.0.8.
- [Release notes](https://github.com/spulec/moto/releases)
- [Changelog](https://github.com/spulec/moto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/spulec/moto/compare/2.0.7...2.0.8)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #667 from hackingmaterials/dependabot/pip/moto-2.0.8

dependabot-preview[bot] committed:
Bump boto3 from 1.17.73 to 1.17.78

Bumps [boto3](https://github.com/boto/boto3) from 1.17.73 to 1.17.78.
- [Release notes](https://github.com/boto/boto3/releases)
- [Changelog](https://github.com/boto/boto3/blob/develop/CHANGELOG.rst)
- [Commits](https://github.com/boto/boto3/compare/1.17.73...1.17.78)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #668 from hackingmaterials/dependabot/pip/boto3-1.17.78

dependabot-preview[bot] committed:
Bump pytest-cov from 2.11.1 to 2.12.0

Bumps [pytest-cov](https://github.com/pytest-dev/pytest-cov) from 2.11.1 to 2.12.0.
- [Release notes](https://github.com/pytest-dev/pytest-cov/releases)
- [Changelog](https://github.com/pytest-dev/pytest-cov/blob/master/CHANGELOG.rst)
- [Commits](https://github.com/pytest-dev/pytest-cov/compare/v2.11.1...v2.12.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #664 from hackingmaterials/dependabot/pip/pytest-cov-2.12.0

utf committed:
Merge pull request #660 from janosh/main

Add common powerup add_metadata

acrutt committed:
Debug GetInsertionCalcs firetasks optimizefw_kwargs so user can provide override_default_vasp_params without conflicting with setting NSW=299 (if NSW is not set by the user)

acrutt committed:
Remove unnecessary import statement

acrutt committed:
Remove automatic setting of NSW=299 since with custodian and the default NSW of 99, ~500 ionic steps should be sufficient

espottesmith committed:
Changes to be compatible with new pymatgen

dependabot-preview[bot] committed:
Bump maggma from 0.27.0 to 0.28.0

Bumps [maggma](https://github.com/materialsproject/maggma) from 0.27.0 to 0.28.0.
- [Release notes](https://github.com/materialsproject/maggma/releases)
- [Changelog](https://github.com/materialsproject/maggma/blob/main/docs/CHANGELOG.md)
- [Commits](https://github.com/materialsproject/maggma/compare/v0.27.0...v0.28.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump boto3 from 1.17.78 to 1.17.84

Bumps [boto3](https://github.com/boto/boto3) from 1.17.78 to 1.17.84.
- [Release notes](https://github.com/boto/boto3/releases)
- [Changelog](https://github.com/boto/boto3/blob/develop/CHANGELOG.rst)
- [Commits](https://github.com/boto/boto3/compare/1.17.78...1.17.84)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump tqdm from 4.60.0 to 4.61.0

Bumps [tqdm](https://github.com/tqdm/tqdm) from 4.60.0 to 4.61.0.
- [Release notes](https://github.com/tqdm/tqdm/releases)
- [Commits](https://github.com/tqdm/tqdm/compare/v4.60.0...v4.61.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #670 from hackingmaterials/dependabot/pip/maggma-0.28.0

dependabot-preview[bot] committed:
Merge pull request #672 from hackingmaterials/dependabot/pip/boto3-1.17.84

dependabot-preview[bot] committed:
Bump coveralls from 3.0.1 to 3.1.0

Bumps [coveralls](https://github.com/TheKevJames/coveralls-python) from 3.0.1 to 3.1.0.
- [Release notes](https://github.com/TheKevJames/coveralls-python/releases)
- [Changelog](https://github.com/TheKevJames/coveralls-python/blob/master/CHANGELOG.md)
- [Commits](https://github.com/TheKevJames/coveralls-python/compare/3.0.1...3.1.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #674 from hackingmaterials/dependabot/pip/tqdm-4.61.0

dependabot-preview[bot] committed:
Merge pull request #673 from hackingmaterials/dependabot/pip/coveralls-3.1.0

mjwen committed:
Correct LREAL and PREC for elastic workflow

utf committed:
Merge pull request #675 from mjwen/elastic

Correct VASP input for elastic workflow

utf committed:
Merge pull request #669 from acrutt/optimizefw_kwargs

Insertion Worflow optimizefw_kwargs

dependabot-preview[bot] committed:
Bump ruamel-yaml from 0.17.4 to 0.17.7

Bumps [ruamel-yaml](https://sourceforge.net/p/ruamel-yaml/code/ci/default/tree) from 0.17.4 to 0.17.7.

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump pytest-cov from 2.12.0 to 2.12.1

Bumps [pytest-cov](https://github.com/pytest-dev/pytest-cov) from 2.12.0 to 2.12.1.
- [Release notes](https://github.com/pytest-dev/pytest-cov/releases)
- [Changelog](https://github.com/pytest-dev/pytest-cov/blob/master/CHANGELOG.rst)
- [Commits](https://github.com/pytest-dev/pytest-cov/compare/v2.12.0...v2.12.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump boto3 from 1.17.84 to 1.17.88

Bumps [boto3](https://github.com/boto/boto3) from 1.17.84 to 1.17.88.
- [Release notes](https://github.com/boto/boto3/releases)
- [Changelog](https://github.com/boto/boto3/blob/develop/CHANGELOG.rst)
- [Commits](https://github.com/boto/boto3/compare/1.17.84...1.17.88)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #677 from hackingmaterials/dependabot/pip/ruamel-yaml-0.17.7

dependabot-preview[bot] committed:
Merge pull request #678 from hackingmaterials/dependabot/pip/pytest-cov-2.12.1

dependabot-preview[bot] committed:
Merge pull request #679 from hackingmaterials/dependabot/pip/boto3-1.17.88

dependabot-preview[bot] committed:
Bump boto3 from 1.17.88 to 1.17.93

Bumps [boto3](https://github.com/boto/boto3) from 1.17.88 to 1.17.93.
- [Release notes](https://github.com/boto/boto3/releases)
- [Changelog](https://github.com/boto/boto3/blob/develop/CHANGELOG.rst)
- [Commits](https://github.com/boto/boto3/compare/1.17.88...1.17.93)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump maggma from 0.28.0 to 0.29.0

Bumps [maggma](https://github.com/materialsproject/maggma) from 0.28.0 to 0.29.0.
- [Release notes](https://github.com/materialsproject/maggma/releases)
- [Changelog](https://github.com/materialsproject/maggma/blob/main/docs/CHANGELOG.md)
- [Commits](https://github.com/materialsproject/maggma/compare/v0.28.0...v0.29.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump monty from 2021.5.9 to 2021.6.10

Bumps [monty](https://github.com/materialsvirtuallab/monty) from 2021.5.9 to 2021.6.10.
- [Release notes](https://github.com/materialsvirtuallab/monty/releases)
- [Changelog](https://github.com/materialsvirtuallab/monty/blob/master/docs/changelog.html)
- [Commits](https://github.com/materialsvirtuallab/monty/compare/v2021.5.9...v2021.6.10)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump ruamel-yaml from 0.17.7 to 0.17.9

Bumps [ruamel-yaml](https://sourceforge.net/p/ruamel-yaml/code/ci/default/tree) from 0.17.7 to 0.17.9.

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #681 from hackingmaterials/dependabot/pip/maggma-0.29.0

dependabot-preview[bot] committed:
Merge pull request #680 from hackingmaterials/dependabot/pip/boto3-1.17.93

dependabot-preview[bot] committed:
Bump moto from 2.0.8 to 2.0.9

Bumps [moto](https://github.com/spulec/moto) from 2.0.8 to 2.0.9.
- [Release notes](https://github.com/spulec/moto/releases)
- [Changelog](https://github.com/spulec/moto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/spulec/moto/compare/2.0.8...2.0.9)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #682 from hackingmaterials/dependabot/pip/monty-2021.6.10

dependabot-preview[bot] committed:
Merge pull request #685 from hackingmaterials/dependabot/pip/ruamel-yaml-0.17.9

dependabot-preview[bot] committed:
Bump tqdm from 4.61.0 to 4.61.1

Bumps [tqdm](https://github.com/tqdm/tqdm) from 4.61.0 to 4.61.1.
- [Release notes](https://github.com/tqdm/tqdm/releases)
- [Commits](https://github.com/tqdm/tqdm/compare/v4.61.0...v4.61.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #683 from hackingmaterials/dependabot/pip/moto-2.0.9

dependabot-preview[bot] committed:
Merge pull request #684 from hackingmaterials/dependabot/pip/tqdm-4.61.1

jmmshn committed:
added citation suggestion

rkingsbury committed:
Merge branch 'main' of https://github.com/hackingmaterials/atomate into qcinput-vdw

dependabot-preview[bot] committed:
Bump maggma from 0.29.0 to 0.29.2

Bumps [maggma](https://github.com/materialsproject/maggma) from 0.29.0 to 0.29.2.
- [Release notes](https://github.com/materialsproject/maggma/releases)
- [Changelog](https://github.com/materialsproject/maggma/blob/main/docs/CHANGELOG.md)
- [Commits](https://github.com/materialsproject/maggma/compare/v0.29.0...v0.29.2)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump boto3 from 1.17.93 to 1.17.97

Bumps [boto3](https://github.com/boto/boto3) from 1.17.93 to 1.17.97.
- [Release notes](https://github.com/boto/boto3/releases)
- [Changelog](https://github.com/boto/boto3/blob/develop/CHANGELOG.rst)
- [Commits](https://github.com/boto/boto3/compare/1.17.93...1.17.97)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump scipy from 1.6.3 to 1.7.0

Bumps [scipy](https://github.com/scipy/scipy) from 1.6.3 to 1.7.0.
- [Release notes](https://github.com/scipy/scipy/releases)
- [Commits](https://github.com/scipy/scipy/compare/v1.6.3...v1.7.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #688 from hackingmaterials/dependabot/pip/boto3-1.17.97

dependabot-preview[bot] committed:
Merge pull request #687 from hackingmaterials/dependabot/pip/maggma-0.29.2

dependabot-preview[bot] committed:
Bump moto from 2.0.9 to 2.0.10

Bumps [moto](https://github.com/spulec/moto) from 2.0.9 to 2.0.10.
- [Release notes](https://github.com/spulec/moto/releases)
- [Changelog](https://github.com/spulec/moto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/spulec/moto/compare/2.0.9...2.0.10)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Merge pull request #689 from hackingmaterials/dependabot/pip/scipy-1.7.0

dependabot-preview[bot] committed:
Merge pull request #690 from hackingmaterials/dependabot/pip/moto-2.0.10

rkingsbury committed:
fix error in QChemDrone vdw changes

rkingsbury committed:
bump pymatgen version

rkingsbury committed:
update KSPACING in test per pymatgen #2163 change

rkingsbury committed:
updated pymatgen version

rkingsbury committed:
Merge branch 'main' into qcinput-vdw

rkingsbury committed:
trivial change to trigger CI

rkingsbury committed:
revert trivial change

janosh committed:
fix ModifyPotcar doc string copy error

rkingsbury committed:
Merge branch 'main' of https://github.com/hackingmaterials/atomate into qcinput-vdw

rkingsbury committed:
Merge branch 'qcinput-vdw' of https://github.com/rkingsbury/atomate into qcinput-vdw

rkingsbury committed:
trivial change

rkingsbury committed:
revert trivial change

espottesmith committed:
Copying Sam's changes for drone, adding ForceFW

computron committed:
Update to v1.0.0

computron committed:
Merge branch 'main' of https://www.github.com/hackingmaterials/atomate into main

computron committed:
Update to v1.0.1

alchem0x2A committed:
Fix bug in VASP IO write when `spec_structure_key` defined 

`WriteVaspFromIOSet` does not correctly update the `vis` when `spec_structure_key` is set. The PR fixes the bug so the user-provided spec key can be passed.

JaGeo committed:
Add optmization to Lobster

JaGeo committed:
Include tests

JaGeo committed:
Add compatibility offline mode

espottesmith committed:
Absolute radical changes to avoid conflicts with pymatgen

JaGeo committed:
add zopen

rkingsbury committed:
Merge branch 'main' of https://github.com/hackingmaterials/atomate into main

rkingsbury committed:
Merge branch 'main' of https://github.com/rkingsbury/atomate into main

rkingsbury committed:
Docstring update (thanks @arosen93)

janosh committed:
ensure use_custodian() does not overwrite modify_gzip_vasp(wf, False) (closes #719)

janosh committed:
add test for use_custodian() following modify_gzip_vasp(wf, False)

janosh committed:
pyupgrade --py37-plus **/*.py + black

janosh committed:
common powerup add_namefile include fw_id

janosh committed:
bump pre-commit hook versions

janosh committed:
pre-commit run black -a

Zhuoying committed:
Merge pull request #691 from rkingsbury/main

update KSPACING in test per pymatgen #2163 change

yang-ruoxi committed:
add absorption

guycmoore committed:
Reorganized the code based on utf's (A. Ganose's) feedback.

guycmoore committed:
Added to docstrings & more tidying up

itsshawnwang committed:
Merge pull request #721 from JaGeo/main

Add additional optimization to Lobster

itsshawnwang committed:
Merge pull request #686 from jmmshn/master

added citation suggestion

Zhuoying committed:
Merge pull request #405 from guymoore13/master

Linear Response Hubbard U Workflow

yang-ruoxi committed:
add absorptionFW

janosh committed:
fix legacy pymongo CI error: AttributeError: 'Cursor' object has no attribute 'count'

janosh committed:
fix legacy pymongo CI error: AttributeError: 'Cursor' object has no attribute 'count'

janosh committed:
refactor MaterialsEhullBuilder.run()

janosh committed:
refactor MaterialsEhullBuilder.run()

janosh committed:
fix flake8 issues: do not use bare except, comparison to None should be 'if cond is None:', format(...) is missing argument(s) for placeholder(s): key, etc.

janosh committed:
fix flake8 issues: do not use bare except, comparison to None should be 'if cond is None:', format(...) is missing argument(s) for placeholder(s): key, etc.

janosh committed:
fix bad import 'from pymatgen import Structure', fix procure_response_dict undeclared variable calcs_skipped=[]

janosh committed:
fix bad import 'from pymatgen import Structure', fix procure_response_dict undeclared variable calcs_skipped=[]

janosh committed:
make all starred imports explicit

janosh committed:
make all starred imports explicit

janosh committed:
fix CI ImportError: cannot import name 'Firework' from 'atomate.vasp.fireworks

janosh committed:
fix CI ImportError: cannot import name 'Firework' from 'atomate.vasp.fireworks

janosh committed:
replace deprecated yaml.safe_load() with yaml.load()

janosh committed:
replace deprecated yaml.safe_load() with yaml.load()

itsshawnwang committed:
Merge pull request #725 from janosh/namefile-add-fwid

add_namefile include firework ID in file name

itsshawnwang committed:
Merge pull request #723 from janosh/main

Ensure use_custodian() does not overwrite modify_gzip_vasp(wf, False)

itsshawnwang committed:
Merge pull request #700 from janosh/main

fix ModifyPotcar doc string copy error

mkhorton committed:
Fix import for pymatgen v2022

mkhorton committed:
Fix import for pymatgen v2022

yang-ruoxi committed:
modified WriteVaspStaticFromPrev

yang-ruoxi committed:
modified WriteVaspStaticFromPrev

Zhuoying committed:
Merge pull request #727 from mkhorton/patch-10

Fix imports for pymatgen v2022

Commit:
pymongo count test

itsshawnwang committed:
Merge pull request #724 from janosh/pyupgrade

janosh committed:
add pyupgrade pre-commit hook and rerun on all files

janosh committed:
add pyupgrade pre-commit hook and rerun on all files

janosh committed:
default to latest circleci/mongo image (3.4.15 was causing pymongo.errors.ConfigurationError: Server at localhost:27017 reports wire version 5, but this version of PyMongo requires at least 6 (MongoDB 3.6))

janosh committed:
default to latest circleci/mongo image (3.4.15 was causing pymongo.errors.ConfigurationError: Server at localhost:27017 reports wire version 5, but this version of PyMongo requires at least 6 (MongoDB 3.6))

janosh committed:
drop outdated docker image materialsvirtuallab/circle-ci-pmg-py3, use pip install pymatgen instead

janosh committed:
revert GridFS.count_documents({}) (doesn't exist) to GridFS.find({}).count()

janosh committed:
revert GridFS.count_documents({}) (doesn't exist) to GridFS.find({}).count()

itsshawnwang committed:
Merge pull request #653 from rkingsbury/qcinput-vdw

support van_der_waals QChem section in write_input

itsshawnwang committed:
Merge pull request #720 from alchem0x2A/patch-1

Fix bug in VASP IO write when `spec_structure_key` defined

JaGeo committed:
Update

JaGeo committed:
Update test_files lobster

JaGeo committed:
Merge branch 'main' of github.com:janosh/atomate

JaGeo committed:
Merge branch 'main' of github.com:hackingmaterials/atomate

JaGeo committed:
small change to lobster workflow, change of validators

janosh committed:
try github CI instead of circleci

janosh committed:
clean up setup.py, add dep openbabel for qchem functionality, test.yml install [complete] deps

janosh committed:
clean up setup.py, add dep openbabel for qchem functionality, test.yml install [complete] deps

itsshawnwang committed:
Merge pull request #731 from JaGeo/main

Fix for test file issue in Lobster workflows

janosh committed:
test.yml snap install openbabel

janosh committed:
remove unused deps from requirements{,-ci}.txt

janosh committed:
remove unused deps from requirements{,-ci}.txt

janosh committed:
Merge remote-tracking branch 'upstream/main' into fix-ci

janosh committed:
sort requirements{,-ci}.txt alphab., use sudo to install openbabel

Zhuoying committed:
Update Dockerfile

Commit:
update numpy

Commit:
Merge branch 'main' of github.com:Zhuoying/atomate

itsshawnwang committed:
Update config.yml

Zhuoying committed:
Update requirements.txt

itsshawnwang committed:
Merge branch 'main' into fix-ci

itsshawnwang committed:
Merge pull request #2 from janosh/fix-ci

Fix ci

Zhuoying committed:
Merge pull request #1 from janosh/fix-ci

Fix ci merge test

itsshawnwang committed:
Update config.yml

Zhuoying committed:
Update config.yml

itsshawnwang committed:
Update test_lobster_tasks.py

itsshawnwang committed:
Update test_lobster_tasks.py

itsshawnwang committed:
Update test_lobster_tasks.py

Zhuoying committed:
Update test_lobster_tasks.py

fix count() caused by pymongo 4.0

Zhuoying committed:
Update test_lobster_tasks.py

GridFs count

Zhuoying committed:
Update test_lobster_tasks.py

Zhuoying committed:
Update test_lobster_tasks.py

Zhuoying committed:
Update test_lobster_tasks.py

Zhuoying committed:
Update config.yml

itsshawnwang committed:
Merge pull request #3 from Zhuoying/main

pull from CI-test-update

itsshawnwang committed:
Update config.yml

itsshawnwang committed:
Update requirements.txt

janosh committed:
add GridFS count fix by @Zhuoying, restore materialsvirtuallab pmg docker image

janosh committed:
pin numpy + pymatgen versions in setup.py + requirements.txt

Zhuoying committed:
Merge pull request #730 from janosh/fix-ci

Fix CI

janosh committed:
add isort commit hook

janosh committed:
isort **/*.py

JaGeo committed:
Change validator

JaGeo committed:
Merge branch 'main' of github.com:hackingmaterials/atomate

itsshawnwang committed:
Merge pull request #733 from JaGeo/main

Change of a validator in Lobster workflows

janosh committed:
fix 5 flake8 F821 undefined name errors

itsshawnwang committed:
Merge branch 'main' into main

yang-ruoxi committed:
incar edit

espottesmith committed:
Merge remote-tracking branch 'hackmat/main' into qchem_update

espottesmith committed:
Uncommenting apparently important function

itsshawnwang committed:
Merge pull request #732 from janosh/isort

Sort imports

espottesmith committed:
Merge remote-tracking branch 'hackmat/main' into qchem_update

espottesmith committed:
Syntax fix

espottesmith committed:
Looks like one merge issue wasn't caught

espottesmith committed:
Bugfix in tests

janosh committed:
test with github action

janosh committed:
pre-commit exclude files in docs/, docs_rst/, test_files/, tests/ directories

janosh committed:
auto-fixes from trailing-whitespace + end-of-file-fixer hooks

janosh committed:
fix broken vasp workflows neb.yaml + spinorbit_coupling.yaml

janosh committed:
fix undeclared logger in atomate/vasp/analysis/linear_response.py, fix ambiguous file names, flake8 ignore E203, W503

janosh committed:
actually let's not exclude files in docs_rst/

janosh committed:
manually link openbabel header files

janosh committed:
fix typos, better doc string format in atomate/qchem/fireworks/core.py

espottesmith committed:
Update custodian version in requirements; fix bug based on pymongo 4.0

itsshawnwang committed:
Merge pull request #593 from espottesmith/qchem_update

Qchem update

itsshawnwang committed:
Merge branch 'main' into random

itsshawnwang committed:
Merge pull request #736 from janosh/random

Squash a few more issues

janosh committed:
remove noop os.path.join() from module_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)))

janosh committed:
remove few more os.path.join() noops

janosh committed:
pyupgrade --py37-plus **/*.py

janosh committed:
black atomate/**/*.py

janosh committed:
isort **/*.py

janosh committed:
remove unused imports

janosh committed:
redo doc string formatting from #736 lost in git merge main

janosh committed:
add mongodb service container to fix ServerSelectionTimeoutError

janosh committed:
pin pymongo to < 4.0.0

janosh committed:
fix bad usage of procure_response_dict by HubbardHundLinRespToDb.run_task(), restore commented-out code deleted in b22f327d

janosh committed:
fix TestInsertionWorkflow (needs general overhaul I think)

janosh committed:
general overhaul of TestInsertionWorkflow

computron committed:
Merge pull request #734 from janosh/github-action

Replace CircleCI with GitHub Action

computron committed:
Update to v1.0.2

janosh committed:
Merge remote-tracking branch 'upstream/main' into stuff

itsshawnwang committed:
Merge pull request #737 from janosh/stuff

Stuff

janosh committed:
drop py37 for compatibility with latest pymatgen

janosh committed:
.circleci/config.yml remove py3devtest, clean up dependency install

janosh committed:
try conda install pymatgen

janosh committed:
remove deprecated atomate.vasp.powerups.add_tags

janosh committed:
replace deprecated mongo db.collection_names() with db.list_collection_names()

janosh committed:
remove deprecated atomate.vasp.powerups.add_namefile

janosh committed:
flynt atomate (auto f-string conversion just to trigger CI run with latest pmg fixes)

janosh committed:
remove no-op string coercion in f"{str(foo)}"

janosh committed:
final batch of f-str conversion

janosh committed:
formatting (just to trigger CI with new pymatgen version)

janosh committed:
fix Lobster tests failing due to "ValueError: INCAR value of LMAXMIX is inconsistent!"

Co-authored-by: J. George <janine.george@bam.de>

janosh committed:
add readme badges for CI status, min supported py version, current PyPI release and monthly downloads

janosh committed:
fix readme logo on pypi page

janosh committed:
set min maggma version to 0.39.0

janosh committed:
setup.py allow latest pymongo (v4) + fix pymongo 3 -> 4 breaking changes

janosh committed:
add optional_param jobs: list[VaspJob] to RunVaspCustodian

janosh committed:
add type hints to DFPTFW

Zhuoying committed:
Merge pull request #738 from janosh/py38+

Drop py37 support

fraricci committed:
fix for contcar.gz not found

itsshawnwang committed:
Merge pull request #741 from janosh/fix-m1-install

Require minimum maggma version 0.39.0

itsshawnwang committed:
Merge pull request #740 from janosh/readme-badges

Readme badges

itsshawnwang committed:
Merge pull request #742 from janosh/run-vasp-custodian

RunVaspCustodian add optional_param jobs: list[VaspJob]

itsshawnwang committed:
Revert "RunVaspCustodian add optional_param jobs: list[VaspJob]"

itsshawnwang committed:
Merge pull request #743 from hackingmaterials/revert-742-run-vasp-custodian

Revert "RunVaspCustodian add optional_param jobs: list[VaspJob]"

computron committed:
Update to v1.0.3

yang-ruoxi committed:
cleanup

itsshawnwang committed:
Merge pull request #726 from yang-ruoxi/absorption

Absorption

fraricci committed:
forgot optional param

itsshawnwang committed:
Merge pull request #745 from fraricci/fix_gz_contcar_issue

Fix gzip contcar issue

janosh committed:
setup.py set long_description_content_type="text/markdown"

janosh committed:
update pre-commit hooks (brings black out of beta)

janosh committed:
apply black changes

itsshawnwang committed:
Merge pull request #746 from janosh/fix-pypi-page

Fix PyPI page

Commit:
merge with upstream
